Plugin/src/main/java/xyz/kyngs/librelogin/velocity/VelocityLibreLogin.java

```diff
@@ -279,4 +283,16 @@ public File getDataFolder() {
     public LibreLoginPlugin<Player, RegisteredServer> getLibreLogin() {
         return this;
     }
+
+    @Nullable
+    @Override
+    public LimboIntegration<RegisteredServer> getLimboIntegration() {
+        if (pluginPresent("nanolimbovelocity") && limboIntegration == null) {
+            limboIntegration = new VelocityNanoLimboIntegration(server,
+                    getConfiguration().get(ConfigurationKeys.LIMBO_PORT_RANGE));
+        }
+        return limboIntegration;
+    }
```

Comment:
I'm not sure whether lazy-loading the integration is a good idea. If I understand correctly, you said that you cannot load the integration before super.enable() due to some configuration issues. Why can't you load it in the overridden enable block after the super.enable() call?

Reply:
We must do this because the super.enable() method registers all limbo servers, and if we register the limboIntegration variable after super.enable(), the console will display the warning "Lobby server/world serverNameHere not found!", and the server will not be registered.